specification/eventhub/resource-manager/Microsoft.EventHub/preview/2023-01-01-preview/namespaces-preview.json

```diff
@@ -0,0 +1,1504 @@
+            },
+            "zoneRedundant": {
+              "type": "boolean",
+              "description": "Enabling this property creates a Standard Event Hubs Namespace in regions supported availability zones."
+            },
+            "encryption": {
+              "$ref": "#/definitions/Encryption",
+              "description": "Properties of BYOK Encryption description"
+            },
+            "privateEndpointConnections": {
+              "description": "List of private endpoint connections.",
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/PrivateEndpointConnection"
+              }
+            },
+            "disableLocalAuth": {
+              "type": "boolean",
+              "description": "This property disables SAS authentication for the Event Hubs namespace."
+            },
+            "alternateName": {
+              "type": "string",
+              "description": "Alternate name specified when alias and namespace names are same."
+            },
+            "geoDataReplication": {
+              "$ref": "#/definitions/GeoDataReplicationProperties",
+              "description": "Geo Data Replication settings for the namespace"
+            }
+          },
+          "description": "Namespace properties supplied for create namespace operation."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/definitions.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Single Namespace item in List or Get Operation"
+    },
+    "Sku": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Name of this SKU.",
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "SkuName",
+            "modelAsString": true
+          }
+        },
+        "tier": {
+          "type": "string",
+          "description": "The billing tier of this particular SKU.",
+          "enum": [
+            "Basic",
+            "Standard",
+            "Premium"
+          ],
+          "x-ms-enum": {
+            "name": "SkuTier",
+            "modelAsString": true
+          }
+        },
+        "capacity": {
+          "format": "int32",
+          "minimum": 0,
+          "type": "integer",
+          "description": "The Event Hubs throughput units for Basic or Standard tiers, where value should be 0 to 20 throughput units. The Event Hubs premium units for Premium tier, where value should be 0 to 10 premium units."
+        }
+      },
+      "required": [
+        "name"
+      ],
+      "description": "SKU parameters supplied to the create namespace operation"
+    },
+    "GeoDataReplicationProperties": {
+      "type": "object",
+      "properties": {
+        "maxReplicationLagDurationInSeconds": {
+          "format": "int32",
+          "type": "integer",
```

Comment:
>type [](http://example.com/codeflow?start=11&length=4)

any min, max range for this?

Reply:
Hey @rkmanda , maxReplicationLagDurationInSeconds holds value 0 or [5min,1day].

Reply:
pl specify it in the swagger